src/main/java/frc/team4909/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,37 @@
+package frc.team4909.robot.subsystems.elevator;
+
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PWMTalonSRX;
+import edu.wpi.first.wpilibj.PWMVictorSPX;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class ElevatorSubsystem extends Subsystem{
+    PWMVictorSPX leftElevatorSPX, rightElevatorSPX;
+    PWMTalonSRX leftElevatorSRX, rightElevatorSRX, pivotSRX;
+    Encoder pivotEncoder, liftEncoder1, liftEncoder2;
+    DigitalInput topLimitSwitch, bottomLimitSwitch;
+
+    public void ElevatorSubsytem(){
+        //Intake Pivot
+        pivotSRX = new PWMTalonSRX(3);
+        pivotEncoder = new Encoder(1, 2);
```

Comment:
we need to test the absolute angle detection mechanisms Monday, but I think that should be a separate subsystem, there's no reason for them to be coupled together

that said, I think we'll end up using a potentiometer or an SRX encoder (abs. angle) if we can stay within the 360 on the output shaft